### DISCUSSION TITLE
Enabling Vercel Skew Protection

### DISCUSSION THREAD

**Discussion by testvinder** [sveltejs/kit#12006] [Q&A]
According to the [Sveltekit docs](https://kit.svelte.dev/docs/adapter-vercel#skew-protection), you need to "visit the Advanced section of your project settings on Vercel." I don't see such a setting on Vercel. In the [Vercel docs on Skew Protection](https://vercel.com/docs/deployments/skew-protection) it seems that Skew protection via svelte.config.js (the only have an example for Next).

Can someone clarify how Skew Protection is enabled after upgrading to adapter-vercel 5.2.0? Thank you!

**Comment by teemingc** [comment]
You may not be able to see it you're not on a paid plan https://vercel.com/docs/accounts/plans/enterprise

> Skew Protection is available in [Beta](https://vercel.com/docs/release-phases#beta) on [Pro and Enterprise plans](https://vercel.com/docs/accounts/plans/enterprise)

**Reply by testvinder** [reply]
Thanks. I'm on the Pro plan. I'll ask Vercel support why I don't see it then.

**Reply by teemingc** [reply]
Sorry I couldn’t help more. I’m on a free plan so I can’t verify this either.

**Reply by testvinder** [reply]
Support replied to me and enabled it for me. So the option can be missing.
